prow/oss/cluster/prow-controller-manager.yaml

```diff
@@ -65,7 +65,7 @@ rules:
     resources:
     - leases
     resourceNames:
-    - prow-sinker-leaderlock
+    - prow-controller-manager-leader-lock
```

Comment:
Does this change go from `leaderlock` to `leader-lock`?

Reply:
/hold

Reply:
Yes, it's a hard coded value here: https://github.com/kubernetes/test-infra/blob/8123e82f3ae29afa5e782ca198f9e303c9035673/prow/cmd/prow-controller-manager/main.go#L145

Reply:
/unhold